app/src/main/java/org/openedx/app/AppAnalytics.kt

```diff
@@ -24,8 +24,19 @@ enum class AppAnalyticsEvent(val eventName: String, val biValue: String) {
         "MainDashboard:Profile",
         "edx.bi.app.main_dashboard.profile"
     ),
+    NOTIFICATION_PERMISSION(
+        "Notification:Setting Permission Status",
+        "edx.bi.app.notification.permission_settings.status"
+    )
 }
 
 enum class AppAnalyticsKey(val key: String) {
     NAME("name"),
+    STATUS("status"),
+}
+
+enum class PermissionStatus(val status: String) {
+    DENIED("denied"),
+    AUTHORIZED("authorized"),
+    NOT_DETERMINED("not_determined"),
```

Comment:
No usage found.

Reply:
for the Android `not_determined` not applicable, cuz `NotificationManagerCompat.from(context).areNotificationsEnabled()` returns boolean. so removed.